tests/test_proxy.py

```diff
@@ -0,0 +1,167 @@
+import json
+import subprocess
+import vcr
+import unittest
+import uuid
+
+from securedrop_proxy import proxy
+from securedrop_proxy import config
+
+
+class TestProxyValidConfig(unittest.TestCase):
+    def setUp(self):
+        self.conf = config.Conf()
+        self.conf.host = 'jsonplaceholder.typicode.com'
+        self.conf.scheme = 'https'
+        self.conf.port = 443
+
+    def on_save(self, fh, res, conf):
+
+        self.fn = str(uuid.uuid4())
+
+        # this will be `qvm-move...` in production
+        subprocess.run(["cp", fh.name, "/tmp/{}".format(self.fn)])
```

Comment:
For these tests I think we can git rid of this line (since I don't see us asserting the presence of the file in `/tmp`, and we'd litter fewer things there during testing!)